node/bft/src/primary.rs

```diff
@@ -1127,6 +1132,7 @@ impl<N: Network> Primary<N> {
                 fast_forward_round = self.storage.increment_to_next_round(fast_forward_round)?;
                 // Clear the proposed batch.
                 *self.proposed_batch.write() = None;
+                self.outstanding_certificate_requests.lock().clear();
```

Comment:
Could we document here why it is safe to clear all outstanding requests, not just the ones for the given round?

It looks like all requests are always from the most recent round. If so, do we need to track the round number in the HashMap?